ci/install-macosx.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "${TRAVIS_OS_NAME}" != "osx" ]; then
```

Comment:
Looks like this file is no longer referenced from the `.travis.yml` so it should be deleted.

Reply:
Done for realsies.